### DISCUSSION TITLE
Is there a Nightly Docker build?

### DISCUSSION THREAD

**Discussion by ItsNoted** [karakeep-app/karakeep#258] [General]
I see updates being made but not to the main Docker build.

**Comment by MohamedBassem** [comment]
The docker tag "latest" gets updated with every commit that lands on the main branch. For example: https://github.com/hoarder-app/hoarder/pkgs/container/hoarder-web/233851943?tag=latest

**Reply by MohamedBassem** [reply]
@ItsNoted does this answer your qusetion?
